swap_meet/vendor.py

```diff
@@ -0,0 +1,115 @@
+from swap_meet.item import Item
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+from swap_meet.clothing import Clothing
+class Vendor:
```

Comment:
Style: Add newlines before the class declaration:

```suggestion
from swap_meet.clothing import Clothing


class Vendor:
```

See: https://www.python.org/dev/peps/pep-0008/#blank-lines